client/src/components/existing-request/existing-request-search.vue

```diff
@@ -10,50 +10,53 @@
       <v-col cols="12" class="red--text" v-if="errorMessage">{{ errorMessage }}</v-col>
 
       <!-- THIRD LINE -->
-      <v-col cols="1" class="ml-n4 mr-4 max-height">
-        <v-img src="../../assets/images/one-icon.png" contain height="60" />
+    </v-row>
+    <v-row align="center" class="mr-4" dense>
+      <v-col cols="1" class="mr-n3">
+        <v-img src="../../assets/images/one-icon.png" contain height="34" />
       </v-col>
       <v-col cols="11" class="max-height">
         <v-text-field :value="existingRequestSearch.nrNum"
-                       @input="setExistingRequestSearch('nrNum', $event)"
-                       filled
-                       validate-on-blur
-                       :rules="nrRules"
-                       placeholder="Business Name or NR Number"
-                       id="nr-num-text-field" />
+                      @input="setExistingRequestSearch('nrNum', $event)"
+                      class="copy-normal"
+                      filled
+                      validate-on-blur
```

Comment:
What does this prop do? Sounds fancy!

Reply:
validate-on-blur?  it prevents the field validation from happening until after the field is removed from the focus so the red error messages don't show up from the first character entered (useful if the field has a length rule for instance)

Reply:
Ah, very cool! Thanks sir